src/api/app/models/event/project.rb

```diff
@@ -18,24 +23,44 @@ class UpdateProjectConfig < Project
     self.raw_type = 'SRCSRV_UPDATE_PROJECT_CONFIG'
     self.description = 'Project _config was updated'
     payload_keys :sender, :files, :comment
+    after_commit :send_to_bus
+
+    def self.message_bus_queue
+      "#{Configuration.amqp_namespace}.project.update_project_conf"
+    end
   end
 
   class UndeleteProject < Project
     self.raw_type = 'SRCSRV_UNDELETE_PROJECT'
     self.description = 'Project was undeleted'
     payload_keys :comment, :sender
+    after_commit :send_to_bus
+
+    def self.message_bus_queue
+      "#{Configuration.amqp_namespace}.project.undelete"
+    end
   end
 
   class UpdateProject < Project
     self.raw_type = 'SRCSRV_UPDATE_PROJECT'
     self.description = 'Project meta was updated'
     payload_keys :sender
+    after_commit :send_to_bus
+
+    def self.message_bus_queue
+      "#{Configuration.amqp_namespace}.project.update"
```

Comment:
These Configuration. calls seem to be pretty expensive. At least with debug on the test instance I see

```
send opensuse.obs.repo.publish_state
  Configuration Load (0.3ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
  Configuration Load (0.2ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
  Configuration Load (0.2ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
send opensuse.obs.repo.publish_state
  Configuration Load (0.2ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
  Configuration Load (0.2ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
  Configuration Load (0.2ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
send opensuse.obs.repo.publish_state
  Configuration Load (0.5ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
  Configuration Load (0.5ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
  Configuration Load (0.5ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
send opensuse.obs.repo.publish_state
  Configuration Load (0.3ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
  Configuration Load (0.2ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
  Configuration Load (0.2ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
send opensuse.obs.repo.publish_state
  Configuration Load (0.3ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
  Configuration Load (0.2ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
  Configuration Load (0.2ms)  SELECT  `configurations`.* FROM `configurations` ORDER BY `configurations`.`id` ASC LIMIT 1
```

Aka it takes ~1 second per event - plus the time to actually send the event

Reply:
aehm, ms of course - ok, not *so* expensive :)